dim_checks/mozdata/fenix/active_users_aggregates/dim_checks.yaml

```diff
@@ -16,7 +16,7 @@ dim_config:
       title: DAU validation - Firefox Android (Fenix)
       description: Validating that DAU sum in Active Users Aggregates is the same as unique client_id count in the live tables (all channels).
       params:
-        condition: dau_sum = distinct_client_count
+        condition: ABS(dau_sum - distinct_client_count) <= 1
```

Comment:
Let's add a comment for future us linking to the [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1836808) so we remember why we set it this way.